src/styles/blocks/catalog.scss

```
@@ -12,14 +12,21 @@
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    margin-bottom: $standard-margin;
+
+    transition: transform 300ms, color 300ms;
```

Create a variable for it & replace in all cases
```suggestion
    transition: transform $transitionDuration, color $transitionDuration;
```